mail-server/monit.nix

```diff
@@ -0,0 +1,38 @@
+#  nixos-mailserver: a simple mail server
+#  Copyright (C) 2016-2017  Robin Raymond
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program. If not, see <http://www.gnu.org/licenses/>
+
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.mailserver;
+in
+{
+  config = lib.mkIf cfg.monitoring {
+    services.monit = {
+      enable = true;
+      config = ''
+        set daemon 120
+        set alert root@localhost
```

Comment:
this should be configurable. Most people running a webserver will not bother checking the root account.

Reply:
Yes it should and I believe I suggested exactly that somewhere but you're right, I might as well implement it in this PR.
This being said I believe you can redirect mails to root to some other account in the postfix config. Doing that would allow all the other things in the system that hardcoded send stuff to root to work, too.